### PR TITLE
Better plot

### PR DESCRIPTION
**Pull request by tgouss**

### CODE REVIEW

File: biorbd_optim/__init__.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def torque_driven(nlp):
         nlp.nu = nlp.u.rows()
 
     @staticmethod
-    def muscles_and_torque_driven(nlp):
+    def muscle_and_torque_driven(nlp):
         dof_names = nlp.model.nameDof()
```
Comment (pariterre) [inline biorbd_optim/__init__.py]: Not sure, I think I prefer with an s

File: biorbd_optim/dynamics.py [hunk 2]
```diff
@@ -18,23 +18,19 @@ def forward_dynamics_torque_driven(states, controls, model):
...
-        muscles_states = []
-
-        for k in range (model.nbMuscleTotal()):
-            muscles_states.append(biorbd.StateDynamics)
-
         q = states[:model.nbQ()]
         qdot = states[model.nbQ():]
+        muscles_states = biorbd.VecBiorbdMuscleStateDynamics(model.nbMuscleTotal())
+        muscles_activations = controls[:model.nbMuscleTotal()]
+        residual_tau = controls[model.nbMuscleTotal():]
```
Comment (pariterre) [inline biorbd_optim/dynamics.py]: Create a nlp in ProblemType

File: biorbd_optim/__init__.py (hunk 1 above)
Comment (tgouss) [reply biorbd_optim/__init__.py]: Done

File: biorbd_optim/dynamics.py (hunk 2 above)
Comment (tgouss) [reply biorbd_optim/dynamics.py]: Done

**Comment by tgouss** [comment]
Ready to merge

**Comment by pariterre** [comment]
LTGM

**Pull request merged**